ansible/roles/etcd/tasks/etcd2-configure.yml

```diff
@@ -11,10 +11,3 @@
 - name: CoreOS | Write etcd config file
   template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
   notify: restart etcd2
```

Comment:
[this] (https://github.com/stephenrlouie/contrib/blob/etcd-ref/ansible/roles/etcd/tasks/install.yml#L7-L8) is also in the same place, etcd.started is not set when this is called